pkgs/games/BeatSaberModManager/default.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+
+  buildDotnetModule,
+  fetchFromGitHub,
+
+  libX11,
+  libICE,
+  libSM,
+  fontconfig,
+}:
+
+buildDotnetModule rec {
+  pname = "BeatSaberModManager";
+  version = "0.0.2";
+
+  src = fetchFromGitHub {
+    owner = "affederaffe";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-6+9pWr8jJzs430Ai2ddh/2DK3C2bQA1e1+BNDrKhyzY=";
+    fetchSubmodules = true; # It vendors BSIPA-Linux
+  };
+
+  nugetDeps = ./deps.nix;
+
+  dotnetInstallFlags = [
+    # FIXME It doesn't like that for some reason beyond my ability to package dotnet
+    "-p:PublishSingleFile=false"
+  ];
```

Comment:
Specifically, it fails with

```
Executing dotnetInstallHook
MSBuild version 17.3.1+2badb37d1 for .NET
  IPA-lib -> /nix/store/hsx50dc5yly2bqinqcdr9sz6xs9zfawc-BeatSaberModManager-0.0.2/lib/BeatSaberModManager/
/nix/store/a5x48hb982y79kkh39n8wh16b62vi1iz-dotnet-sdk-6.0.401/sdk/6.0.401/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Publish.targets(102,5): error NETSDK1097: It is not supported to publish an application to a single-file without specifying a RuntimeIdentifier. You must either specify a RuntimeIdentifier or set PublishSingleFile to false. [/build/source/BeatSaberModManager/BeatSaberModManager.csproj]
```

If anyone more knowledgable in dotnet knows how to fix this without disabling single-file, let me know.

Reply:
Found a solution but it's not pretty.

Reply:
Created an issue https://github.com/affederaffe/BeatSaberModManager/issues/5